setup.py

```diff
@@ -103,6 +103,7 @@ def run(self):
         "pygments",
         "pyyaml",
         "pillow",
+        "rosbags",
```

Comment:
Thank you for making the ros2 bag version. This is exactly what I wanted

With an environment with an old `rosbags` installed I got this error message because `Reader.messages` required `topics` as a keyword argument instead of `connections`, so I'd like to propose to specify the package version. 

```TypeError: messages() got an unexpected keyword argument 'connections'```

The problem is that `rosbags` doesn't explicitly manage the package versions on pip, so I think we need to specify the git URL like 

```"rosbags@git+https://gitlab.com/ternaris/rosbags.git#c9bd6ebbc02d14dff16883481aa101e9116bf898",```

Reply:
There are versions that can be used by pip, e.g. `pip install "rosbags>=0.9.8"` works. So we can specify `rosbags>=0.9.8` here.